library/hkdf.c

```diff
@@ -18,13 +18,13 @@
  */
 #include "common.h"
 
-#if defined(MBEDTLS_HKDF_C)
-
 #include <string.h>
 #include "mbedtls/hkdf.h"
 #include "mbedtls/platform_util.h"
 #include "mbedtls/error.h"
 
+#if defined(MBEDTLS_PSA_BUILTIN_ALG_HKDF)
```

Comment:
This file is a non-PSA low-level crypto module. It shouldn't have any reference to PSA.